examples/chart/teleport-cluster/values.yaml

```diff
@@ -37,6 +37,21 @@ proxyListenerMode: ""
 separatePostgresListener: false
 separateMongoListener: false
 
+# Public cluster address, including port. Defaults to `clusterName` on 443 port.
+publicAddress: ""
+# Public cluster SSH address, including port. Defaults to `publicAddress` on 3023 port.
+# Only used when `proxyListenerMode` is not 'multiplex'.
+sshPublicAddress: ""
+# Public cluster tunnel SSH address, including port. Defaults to `publicAddress` on 3024 port.
+# Only used when `proxyListenerMode` is not 'multiplex'.
+tunnelPublicAddress: ""
+# Public cluster postgres listener address, including port. Defaults to `publicAddress` on 5432 port.
+# Only used when `proxyListenerMode` is not 'multiplex' and `separatePostgresListener` is true.
+postgresPublicAddress: ""
+# Public cluster mongo listener address, including port. Defaults to `publicAddress` on 27017 port.
+# Only used when `proxyListenerMode` is not 'multiplex' and `separateMongoListener` is true.
+mongoPublicAddress: ""
```

Comment:
We also have `mysql_public_addr`, might as well include it too.

Reply:
Done, I also added `kube_proxy_address`